cdap-runtime-ext-dataproc/pom.xml

```diff
@@ -37,6 +37,11 @@
       <version>${project.version}</version>
       <scope>provided</scope>
     </dependency>
+    <dependency>
+      <groupId>io.cdap.cdap</groupId>
+      <artifactId>cdap-common</artifactId>
```

Comment:
No. Don't add common dependency. It should only depends on the `cdap-runtime-spi`

Reply:
removed